src/App.css

```diff
@@ -1,28 +1,28 @@
+@import url("https://fonts.googleapis.com/css?family=Nunito:600,700");
+body {
+  font-family: 'Nunito', sans-serif; }
+
 .App {
-  text-align: center;
-}
+  text-align: center; }
 
 .App-logo {
   animation: App-logo-spin infinite 20s linear;
-  height: 80px;
-}
+  height: 80px; }
 
 .App-header {
   background-color: #222;
   height: 150px;
   padding: 20px;
-  color: white;
-}
+  color: white; }
```

Comment:
There's a saying in the UI world. 
`Manners may maketh a man, but it's indentation which makes a developer!`

Indent properly, move the brackets to proper places and not just your end product, but your code will look beautiful as well! 😄

Reply:
@divyamrastogi Sorry, I cannot fix this, since this is happening with every SCSS file being converted into CSS.

Reply:
You're right. Everyone was facing that issue. I've added App.css to `.gitignore` now. Just take a pull from master and you won't have to commit your `App.css`.

Reply:
@divyamrastogi No, it isn't getting ignored. I think you should change it to src/App.css